cardano-cli/src/Cardano/CLI/EraBased/Run/Governance/Actions.hs

```diff
@@ -502,3 +513,22 @@ runGovernanceActionHardforkInitCmd
     firstExceptT GovernanceActionsCmdWriteFileError . newExceptT $
       conwayEraOnwardsConstraints eon $
         writeFileTextEnvelope outFile (Just "Hardfork initiation proposal") proposalProcedure
+
+carryHashChecks
+  :: MustCheckHash a
+  -> L.Anchor L.StandardCrypto
+  -> AnchorDataTypeCheck
+  -> ExceptT GovernanceActionsError IO ()
+carryHashChecks checkHash anchor errorAdaptor =
```

Comment:
This `errorAdaptor` name is a bit confusing here. It's not adapting anything, but it's more of a type of check. So maybe:
```suggestion
carryHashChecks checkHash anchor checkType =
```
?

Reply:
Addressed here: https://github.com/IntersectMBO/cardano-cli/pull/915/commits/9aae3ce0cc4720cedc126945ad7f5fea1c4d8f49